### ISSUE TITLE
New accounts type needs to be added

### ISSUE THREAD

**Issue by mattu08** [kevbite/CompaniesHouse.NET#147]
Hi,

Just started getting the following error on "last accounts";

```
 "last_accounts": {
      "type": "no-accounts-type-available"
    },
```

Then realized that the type was missing, could you add the missing account type:

`no-accounts-type-available`

To /src/CompaniesHouse/Response/CompanyProfile/LastAccountsType.cs

https://github.com/kevbite/CompaniesHouse.NET/blob/efef661d99dc64b38ed560e6d908695649f3ac1e/src/CompaniesHouse/Response/CompanyProfile/LastAccountsType.cs

Cheers,

Matt

**Comment by kevbite** [comment]
Hey @mattu08, thanks for raising this issue, I'll try to get it sorted as soon as possible.

**Comment by kevbite** [comment]
This is now fixed in the latest version [7.9.2](https://www.nuget.org/packages/CompaniesHouse/7.9.2).

**Comment by mattu08** [comment]
Thanks @kevbite 👍
